tox.ini

```diff
@@ -16,6 +16,15 @@ deps =
 commands =
     bash -c "bashate **/*.sh"
 
+[testenv:lint-yaml]
+whitelist_externals = bash
+basepython = python3.6
```

Comment:
Do you need to specify `basepython`? It's not inherited from `testenv`?

Reply:
Fixed